tests/symengine/symengine_tensor_operations_02.output

```diff
@@ -92,13 +92,13 @@ DEAL:Float:Math operators:Tensor::c+=x: 9.5 + x_00 11.0 + x_01 12.5 + x_10 14.0
 DEAL:Float:Math operators:Tensor::c*=y: y*(9.5 + x_00) y*(11.0 + x_01) y*(12.5 + x_10) y*(14.0 + x_11)
 DEAL:Float:Math operators:Tensor::c*=s: 2.0*y*(9.5 + x_00) 2.0*y*(11.0 + x_01) 2.0*y*(12.5 + x_10) 2.0*y*(14.0 + x_11)
 DEAL:Float:Math operators:Tensor::c-=x: -x_00 + 2.0*y*(9.5 + x_00) -x_01 + 2.0*y*(11.0 + x_01) -x_10 + 2.0*y*(12.5 + x_10) -x_11 + 2.0*y*(14.0 + x_11)
-DEAL:Float:Math operators:Tensor::c/=y: (-x_00 + 2.0*y*(9.5 + x_00))/y (-x_01 + 2.0*y*(11.0 + x_01))/y (-x_10 + 2.0*y*(12.5 + x_10))/y (-x_11 + 2.0*y*(14.0 + x_11))/y
+DEAL:Float:Math operators:Tensor::c/=y: 1.0*(-x_00 + 2.0*y*(9.5 + x_00))/y 1.0*(-x_01 + 2.0*y*(11.0 + x_01))/y 1.0*(-x_10 + 2.0*y*(12.5 + x_10))/y 1.0*(-x_11 + 2.0*y*(14.0 + x_11))/y
```

Comment:
Has this been generated with a new version of SymEngine? They're forever changing their printed output -- its impossible to maintain these tests!